src/lib/Form/DataTransformer/VersionInfoTransformer.php

```diff
@@ -9,36 +9,25 @@
 namespace EzSystems\EzPlatformAdminUi\Form\DataTransformer;
 
 use eZ\Publish\API\Repository\ContentService;
-use eZ\Publish\API\Repository\Exceptions\NotFoundException;
-use eZ\Publish\API\Repository\Exceptions\UnauthorizedException;
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
 use eZ\Publish\API\Repository\Values\Content\VersionInfo;
 use Symfony\Component\Form\DataTransformerInterface;
 use Symfony\Component\Form\Exception\TransformationFailedException;
 
 /**
  * Translates Content's ID to domain specific VersionInfo object.
  */
-class VersionInfoTransformer implements DataTransformerInterface
+final class VersionInfoTransformer implements DataTransformerInterface
 {
-    /** @var ContentService */
-    protected $contentService;
+    /** @var \eZ\Publish\API\Repository\ContentService */
+    private $contentService;
 
-    /**
-     * @param ContentService $contentService
-     */
     public function __construct(ContentService $contentService)
     {
         $this->contentService = $contentService;
     }
 
-    /**
-     * @param VersionInfo|null $value
```

Comment:
Cannot be dropped on our end since we cannot define strict method prototype here.

Reply:
Fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1148/commits/42437d627f79cccd4225b508580a7512e00f0d8d